utils/fetches.js

```diff
@@ -100,6 +100,50 @@ const blogPage = {
   },
 };
 
+const campaignListingPage = {
+  serverSideProps: async (context) => {
+    const uri = [];
+    if (context?.query.page) {
+      uri.push(`page=${get(context, "query.page", 1)}`);
+    }
+    if (context?.query.perPage) {
+      uri.push(`perPage=${get(context, "query.perPage", "")}`);
+    }
+    const data = await fetch(
+      `${reqGetHost(context?.req)}/api/lib/v1/campaigns?${uri.join("&")}`
+    ).then((result) => result.json());
+    return {
+      props: data,
+    };
+  },
+};
```

Comment:
detto, ak je to specificke pre euronics, tak aj tieto fetche by mali ostat na spodu konkretnej stranky. Ak mas ale v plane kampane dat do default (**co by podla mna jednoznacne mali, preto to mergnem**) tak je to ok